diff --git a/board/drivers/can_common.h b/board/drivers/can_common.h
index fd2cfad4..b70203f7 100644
--- a/board/drivers/can_common.h
+++ b/board/drivers/can_common.h
@@ -208,6 +208,11 @@ void ignition_can_hook(CANPacket_t *to_push) {
       ignition_can_cnt = 0U;
     }

+    // Volkswagen MEB exception
+    if ((addr == 0x3C0) && (len == 4)) {
+      ignition_can = GET_BIT(to_push, 17);
+      ignition_can_cnt = 0U;
+    }
   }
 }